sys-libs/tree-standalone/tree-standalone-1.0.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Compatibility library for musl"
+HOMEPAGE="https://github.com/void-linux/void-packages/tree/master/srcpkgs/musl-legacy-compat"
+
+LICENSE="|| ( BSD-2 BSD )"
+SLOT="0"
+KEYWORDS="~amd64"
+SRC_URI="https://raw.githubusercontent.com/void-linux/void-packages/master/srcpkgs/musl-legacy-compat/files/tree.h"
+
+S="${WORKDIR}"
+
+src_unpack() {
+	cp "${DISTDIR}/${PN%%-*}.h" "${WORKDIR}"
```

Comment:
|| die

Reply:
Do you prefer this over just doing doheader from DISTDIR in src_install()?

and then empty src_unpack so it doesn't try to unpack stuff.